### PR TITLE
add-sign-in

### PR DESCRIPTION
**Pull request by Ecne26**

### CODE REVIEW

**Review by lerik1408 — APPROVED** [review]

**Review by lerik1408 — APPROVED** [review]
Good Job!
BTW you have conflicting files

**Review by BillyAleks — CHANGES_REQUESTED** [review]
Thank you, please check the comments. Remove React imports along your changes. And consider using Yup and Formik for form hanling and validation.

**Review by lerik1408 — COMMENTED** [review]
GJ, but you have conflict

File: src/components/ChangePassword.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+export default function ChangePassword() {
+    const [shouldRedirect, setShouldRedirect] = useState(false);
+    const [successAlert, setSuccessAlert] = useState(false);
+    const timer = useRef(null);
+
+    useEffect(() => {
+        return () => clearTimeout(timer.current);
+    }, []);
+
+    const validationSchema = yup.object({
```
Comment (BillyAleks) [inline src/components/ChangePassword.js:25]: I would recommend to write validation schema in separate module

**Review by BillyAleks — APPROVED** [review]
All is good, just consider moving validation to separate modules.

**Pull request merged**